.github/workflows/reusable.yml

```diff
@@ -24,6 +24,9 @@ on:
       extra_envs:
         required: false
         type: string
+      extra_run_command:
```

Comment:
can we merge `extra_run_command` and `extra_build_args` into one parameter?

Reply:
OK.